plugins/conform

```diff
@@ -0,0 +1 @@
+repository = https://github.com/skyzyx/asdf-conform
```

Comment:
This should be the URL to the Git repository itself, which for GitHub project would be `https://github.com/skyzyx/asdf-conform.git`.